styleguide/derek/pattern-components/customer-logos/customer-logos.ejs

```diff
@@ -0,0 +1,12 @@
+<div class="customerLogos-wrapper">
+  <div class="customerLogos-container">
+    <div class="customerLogos-withCallout">
+      <div class="customerLogos-imgRow clearfix">
+      <%- partial('_partials/customer-logo.ejs') %>
+      <%- partial('_partials/customer-logo.ejs', {img: 'https://0fb555c65f288d09a992-a2085a232a1d2a458a641f48ff12bf08.ssl.cf2.rackcdn.com/LP-assets/CSID.png'}) %>
+      <%- partial('_partials/customer-logo.ejs', {img: 'https://0fb555c65f288d09a992-a2085a232a1d2a458a641f48ff12bf08.ssl.cf2.rackcdn.com/LP-assets/Pipedrive.png'}) %>
+      <%- partial('_partials/customer-logo.ejs', {img: 'https://0fb555c65f288d09a992-a2085a232a1d2a458a641f48ff12bf08.ssl.cf2.rackcdn.com/LP-assets/Under-Armor.png'}) %>
```

Comment:
same here

Reply:
![image](https://user-images.githubusercontent.com/5263371/29939215-ba127492-8e50-11e7-9316-8f4e1dd380d2.png)

The under armor logo here looks funky. Did the text get chopped off?

Reply:
These logos are just stub content. They arent super optimized. I think it would be easier to replace these with more permanent versions if they live in the CDN?

Reply:
Images ultimately need to be sourced from rackspace.com not the CDN. This is part of our performance initiative, and will definitely come into play once we're on akamai.

So if we're dealing with general content types, which don't have a form upload field, then yes it needs to be in the zoolander repo. This is something that can get resolved with the media library functionality in D8, but for D7 it's gonna have to live in zoolander.

Reply:
On second thought, we may be able to use the image content type for this stuff.  Just use that to upload the images and then we could reference those images inside the general.